xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -90,7 +91,7 @@ class CGUIDialogVideoInfo :
    * \param pItem Search result item
    */
   void OnSearchItemFound(const CFileItem* pItem);
-  void OnManageVideoVersions();
+  bool OnManageVideoVersions();
   void OnManageVideoExtras();
```

Comment:
Should this be changed in the same way we changed `OnManageVideoVersions` in this PR, adding a return value and using it to refresh properly?

Reply:
I don't think that's needed now, with extras not exposed in the library, the refresh would never be useful.

The only situation I can think of where it would make a difference is the addition of a library item (a movie) as an extra of another movie (possible through file browser). A lack of refresh would be a minor inconvenience compared with the db corruption that would happen... I'll add a validation to avoid if there isn't one already.

If extras were displayed in a library subfolder of movies in the future, then it would make sense to add, but likely only for specific navigation and different requirements than versions.

Reply:
Alright, understood. Then there is no need to add add this now.